### ISSUE TITLE
Enhancement: refresh note after change of label "template"

### ISSUE THREAD

**Issue by rzw** [zadam/trilium#261]
I discriminate two types of tasks (with and without due date). My opening page has a relation definition for a template. After choosing one of (in my case) two templates hitting ENTER will not update the note. I can use F5 or (using the mouse) click on type and keep text. I then have different labels in the UI. E.g. without due date a boolean done and multi value tag field.

Long story short: an instant change of the labels in the UI according to the chosen template would be cool even though hitting F5 isn't a big deal. I'm not aware of any draw backs but you, Adam, would know better.

**Comment by zadam** [comment]
Yes, I noticed this as well few times that it might be useful to automatically refresh the note detail after changing attributes. It shouldn't be difficult to change this ...
